tools/mini-yetus.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
I didn't see any "bashism" in the script to enforce bash... better to keep `/bin/sh`....

Reply:
Are you bashing my script? may the `/bin/sh` be with you!